IamCredentials/owlbot.py

```diff
@@ -57,16 +50,6 @@
     r"\$transportConfig, and any \$serviceAddress",
     r"$transportConfig, and any `$apiEndpoint`")
 
-# fix year
-s.replace(
-    '**/*Client.php',
-    r'Copyright \d{4}',
-    'Copyright 2021')
-s.replace(
-    'tests/**/*Test.php',
-    r'Copyright \d{4}',
-    'Copyright 2021')
-
 # Change the wording for the deprecation warning.
 s.replace(
     'src/*/*_*.php',
```

Comment:
Can we go ahead and make the change like in the other beta APIs and remove the `_` proto files please?